src/Package.php

```diff
@@ -39,6 +41,13 @@ public static function init() {
 
 		// Avoid double initialization when the feature plugin is in use.
 		if ( defined( 'WC_ADMIN_VERSION_NUMBER' ) ) {
+			$update_version = new WC_Admin_Notes_Update_Version();
+			if ( version_compare( WC_ADMIN_VERSION_NUMBER, self::VERSION, '<' ) ) {
+				$update_version::add_note();
+			} else {
+				$update_version::delete_note();
```

Comment:
Why not just have the `if` statement and not create the note at all in the `else` case.

Reply:
The user could update WCA to the current version. Without the delete_note, the notice would continue to display.